libs/cdk/src/lib/zone-configuration/event-names.ts

```diff
@@ -156,8 +167,14 @@ export const websocketEvents: (keyof WebSocketEventMap)[] = [
   'open',
 ];
 
-// XHREvent (group is here to make it easier to target HXR in angular applications. It contains redundant events e.g. 'error')
-export const xhrEvents = ['XHR', 'load', 'error'];
+/**
+ * Basic XHR Events
+ * [Load](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/load_event)
+ * [Error](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/error_event)
+ * There are more events you may want to unpatch https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest
+ */
+// XHREvent (group is here to make it easier to target XHR in angular applications. It contains redundant events e.g. 'error')
+export const xhrEvents = ['load', 'error'];
```

Comment:
@BioPhoton I checked, there is no `XHR` event in MDN, also when request is sent using HttpClient, only two events get subscribed on – `load` and `error`, so I removed `XHR` from here

Reply:
👍